src/modules/gnd_pos_control/GroundRoverPositionControl.cpp

```diff
@@ -135,11 +124,7 @@ GroundRoverPositionControl::GroundRoverPositionControl() :
 	_parameter_handles.l1_damping = param_find("GND_L1_DAMPING");
 	_parameter_handles.l1_distance = param_find("GND_L1_DIST");
 
-
-	_parameter_handles.airspeed_min = param_find("FW_AIRSPD_MIN");
-	_parameter_handles.airspeed_trim = param_find("FW_AIRSPD_TRIM");
 	_parameter_handles.airspeed_max = param_find("FW_AIRSPD_MAX");
```

Comment:
My assumption was the car would be too close to the ground. For me this will definitely be an outside toy.